src/odm_sharing/tools/share.py

```diff
@@ -167,13 +166,7 @@ def share(
     in_name = Path(input).stem
 
     if outfmt == OutFmt.AUTO:
-        fmt = infer_outfmt(input)
-        if not fmt:
-            logging.warning('unable to infer output format from input path, ' +
```

Comment:
I think its nice to have this logging but rather than a warning have it be an info.

Reply:
well, at this point there is no logic supporting that since `infer_outfmt` always returns a valid format. If an invalid url is given then it would trigger the exception "unrecognized data source format" in cons.py.

Reply:
I meant if the user does not provide an output format, we should let them know what output format we've chosen for them and why.

Reply:
ah ok. is "info: defaulting to excel output format" good?

Reply:
yes, so it would be like this: `info: defaulting to <inferred-format-name> output format`

Reply:
or do you want it more like: `info: using <inferred-format-name> as output format`?

Reply:
done